packages/terra-toggle-section-header/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "terra-toggle-section-header",
+  "main": "lib/ToggleSectionHeader.js",
+  "version": "0.0.0",
+  "description": "Toggle section header component that transitions content in and out with a click.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-toggle-section-header",
+    "ToggleSectionHeader",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0",
+    "terra-mixins": "^1.13.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "react": "^16.2.0",
```

Comment:
We don't want to specify react tor react-dom here.

Reply:
Did this to avoid 
![image](https://user-images.githubusercontent.com/17390173/37349617-d824cad8-26a4-11e8-962b-99f14b3c4ae4.png)
If we want to leave them out and get the warnings, I'm fine updating to remove them both

Reply:
These should be installed at the application level, not per component level. This removes the off changes that mis-matched react/react-dom dependencies are installed.

Reply:
We don't want to include react or react-dom as dependencies, we leave those as peerDeps in all our components to allow more control to consumers on which version they use.

Reply:
Removed here 4e7cf752d5d856eaf3d7b3c3d6fc363fd6780dd4